howto/install/deploy-juju.md

```diff
@@ -1,73 +1,67 @@
-With this quick start guide and some tools from Canonical, you'll have an Anbox Cloud running on the cloud of your choice in minutes!
+Anbox Cloud supports various public clouds, such as AWS, Azure and Google. To deploy Anbox Cloud in a cloud environment, you use Juju.
```

Comment:
Is it correct that Azure is supported? It was mentioned below, but it isn't in the prerequisites.